logzio_alerts_client/logzio_alerts_client_test.go

```diff
@@ -0,0 +1,81 @@
+package logzio_alerts_client
+
+import (
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/model/rulefmt"
+	"gopkg.in/yaml.v3"
+	"os"
+	"reflect"
+	"testing"
+	"time"
+)
+
+func generateTestLogzioGrafanaAlertsClient() *LogzioGrafanaAlertsClient {
+	logzioUrl := os.Getenv("LOGZIO_API_URL")
+	logzioAPIToken := os.Getenv("LOGZIO_API_TOKEN")
+	rulesDS := os.Getenv("RULES_DS")
+	logzioGrafanaAlertsClient := NewLogzioGrafanaAlertsClient(logzioUrl, logzioAPIToken, rulesDS, "integration-test")
+	return logzioGrafanaAlertsClient
+
+}
+
+func TestGenerateGrafanaAlert(t *testing.T) {
+	cl := generateTestLogzioGrafanaAlertsClient()
+	// Define common rule parts for reuse in test cases
+	baseRule := rulefmt.RuleNode{
```

Comment:
Do we want to add a negative test, to see that it fails if we get an invalid rule?

Reply:
added